deps/3rdparty/cassowary-0.60/cassowary-0.60-Windows.vcxproj

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project DefaultTargets="Build" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
>Project [](start = 1, length = 7)

I don't think its a bad thing to do but updating the 3rdparty deps looks like a separate change to me? At least would be easier if we need to revert to avoid shoving too much in this review?

Reply:
Yes, I'm reverting this to keep the changes more incremental.